ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-commands.sh

```diff
@@ -36,8 +36,8 @@ function post-ocp-66839(){
     fi
     # New gained cap annotation should be in extracted creds 
     newCap=$(grep -rh "capability.openshift.io/name:" "${credsDir}"|awk -F": " '{print $NF}'|sort -u|xargs)
-    if [[ "${newCap}" != "MachineAPI ImageRegistry" ]] && [[ "${newCap}" != "ImageRegistry MachineAPI" ]]; then
-        echo "Tobe gained CRs with cap annotation: ${newCap}, but expected: MachineAPI and ImageRegistry"
+    if [[ "${newCap}" != "${EXPECTED_CAPABILITIES_IN_CREDENTIALREQUEST}" ]]; then
```

Comment:
can we do some similar sort for `EXPECTED_CAPABILITIES_IN_CREDENTIALREQUEST` before the comparision.

Reply:
updated